app/alarm/ui/src/main/java/org/phoebus/applications/alarm/ui/table/AlarmTableUI.java

```diff
@@ -7,9 +7,12 @@
  *******************************************************************************/
 package org.phoebus.applications.alarm.ui.table;
 
+import java.rmi.activation.Activator;
```

Comment:
This may be a bogus import, we're not using RMI

Reply:
If we _do_ use RMI Activation, we should reconsider, since it's deprecated, https://openjdk.java.net/jeps/385

Reply:
Fixing this now

Reply:
It was a bogus import and it has been removed